runtime.md

```diff
@@ -13,6 +13,8 @@ The state of a container MUST include, at least, the following properties:
 * **`id`**: (string) is the container's ID.
 This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
+The ID is provided in the state because hooks will be executed with the state as the payload.
```

Comment:
we don't need to hijack the id's description to tell someone why its here because of another feature.  It is enable to document that in the hooks description that the container's state is given to the hook as it's payload.